Formula/duckdb.rb

```diff
@@ -17,21 +19,27 @@ class Duckdb < Formula
 
   depends_on "cmake" => :build
   depends_on "python@3.9" => :build
-  depends_on "utf8proc"
 
   def install
-    ENV.deparallelize if OS.linux? # amalgamation builds take GBs of RAM
-    mkdir "build/amalgamation"
+    args = %w[
+      BUILD_ICU=1
```

Comment:
Does this vendor its own ICU?

Reply:
It isn't actually libicu, but rather they flip on their own utf8proc which exists in https://github.com/duckdb/duckdb/tree/master/third_party/utf8proc. 

Duckdb is built to be self-contained with no external dependencies beyond the c library.

Reply:
This should be using Homebrew `icu4c` then.

Reply:
Unfortunately, it is a modified and stripped-down version of icu. Homebrew's `icu4c` is not a drop-in replacement for it.